src/index.ts

```diff
@@ -278,6 +303,15 @@ export async function install(
   };
   const packageBundle = await rollup.rollup(inputOptions);
   await packageBundle.write(outputOptions);
+  if (isExplicit) {
```

Comment:
can remove this check as well, isExplicit is already handled above (`styles` would always be an empty array).